internal/provider/resource_streaming_namespace.go

```diff
@@ -126,6 +126,12 @@ func (r *StreamingNamespaceResource) Create(ctx context.Context, req resource.Cr
 	if resp.Diagnostics.HasError() {
 		return
 	}
+	// if we have no errors, but the policies is null, we need to remove the namespace from the state
+	if policiesFromServer.IsNull() {
+		// remove it from the state
+		resp.State.RemoveResource(ctx)
+		return
+	}
```

Comment:
Is this correct?  It seems like the policies should never be null immediately after creating the namespace.

Reply:
This is the case with my changes in the PR [here](https://github.com/datastax/terraform-provider-astra/pull/425/files#diff-f8f9c7850ffbed4a8ac84833d849e0d4662a7ce4a69742ef5ec249540285ea75R272-R276)